src/packages/NUnit.2.6.4/lib/nunit.framework.xml

```diff
@@ -0,0 +1,10984 @@
+<?xml version="1.0"?>
+<doc>
```

Comment:
Was `NUnit.2.6.4` folder added by mistake?

Reply:
That was probably not the intention. The only (kind-of) explanation I may have, is that when I tried to clone Reko repository on a new machine, the nunit.framwork.dll was missing and was unable to execute a vast majority of the tests. Probably the result of a NuGet action.

Reply:
We should consider adding Nunit as a nuget package though. I can tinker with this this evening

Reply:
> That was probably not the intention.

So it should be removed. It works fine without that directory on my machine (and on the `Travis`/'Appveyor')